desktop/apps/auction2/components/basic_checkbox/index.js

```diff
@@ -0,0 +1,24 @@
+import { default as React, PropTypes } from 'react';
+
+export default function BasicCheckbox({ item, onClickAction, checked, onClick }) {
+  return (
+    <div className='auction2-basic-checkbox'>
+      <div className='artsy-checkbox' onClick={() => onClick(item.id)}>
+        <div className='artsy-checkbox--checkbox'>
+          <input
+            type='checkbox'
+            name='artistAggregation'
+            value={item.id}
+            checked={checked}
+            onChange={() => {}} // necessary to get around warning
```

Comment:
React throws an error when you set a value on an input without an `onChange` event. Since I'm reusing the artsy checkbox component styles, the `onClick` is actually on the parent element, not on the input itself.

It seems like a valid (albeit different) use of the input component, but this seemed like the best way to get around the annoying warning. (i.e. the suggestion to make it a `readOnly` component makes it _actually_ read only, which is not something we want).